index.html

```diff
@@ -52,13 +52,14 @@
       </nav>
       <div class="hero-image center">
           <div class="text-wrapper">
-            <h1>Tour of Cambridge</h1>
+            <h1>Walking Tours of Cambridge</h1>
```

Comment:
This was intentionally set to `Tour of Cambridge` to help SEO. More people search for the term tour of cambridge than walking tours of cambridge so changing this will be worse for the search engine rankings

Reply:
I've had a chat with mum about this and she says she would prefer for it to be 'Walking Tours of Cambridge' as that's more descriptive of the product and makes better grammatical sense, even if it does dent the SEO a bit.